File: AngularClientApp/src/app/models/dtos/traceInformationDto.ts
import { TraceInfoTableItem } from '../underlying/traceInfoTableItem';

export class TraceInformationDto {
  traceTitle: string;
  port: string; // for trace on bop use bop's serial plus port number "879151-3"
  rtuTitle: string;

  equipment: TraceInfoTableItem[];
  nodes: TraceInfoTableItem[];

  isLightMonitoring: boolean;
  comment: string;
}

File: Shared/Dto/ForWebApi/Trace/TraceInformationDto.cs
using System.Collections.Generic;

namespace Iit.Fibertest.Dto
{
    public class TraceInformationDto
    {
        public string TraceTitle;

        // for detached trace "-1";
        // for trace on bop use bop's serial plus port number "879151-3"
        public string Port; 
        
        public string RtuTitle;

        public List<TraceInfoTableItem> Equipment;
        public List<TraceInfoTableItem> Nodes;

        public bool IsLightMonitoring;
        public string Comment;
    }
}